### DISCUSSION TITLE
Documentation contribution

### DISCUSSION THREAD

**Discussion by giovannicimolin** [hyochan/react-native-iap#1365] [General]
Hey! 

Recently, I've started using this library to add IAP functionality to an app.
I noticed the lack of a comprehensive documentation, and I think I can help you guys out.

If there's someone with merge permissions interested in reviewing my work, can you create a new GitHub project related to documentation? 

I'm happy to contribute with these changes.

**Comment by hyochan** [comment]
@giovannicimolin Very happy to see you giving an interest in documentation. Could you start by forking the `repo` in first hand without a `write` permission?

I am giving a `write` permission to contributors who've spent much time previously in principle.

If you want to host your document, we can make this project mono repo as we did in [hackatalk](http://github.com/dooboolab/hackatalk). We maintain [our doc](https://website.hackatalk.dev) in the same repository.

**Reply by giovannicimolin** [reply]
@hyochan 
> Could you start by forking the repo in first hand without a write permission?

Yup, this was my idea. I never wanted merge rights, just checking if there was interest in documentation from the repo owners. :)
I'll start some basic documentation and contribute it with a PR.

Btw, this is the first I've heard of Docusaurus, and it looks awesome, I'm going to use for this contribution.

**Reply by giovannicimolin** [reply]
@hyochan I opened a PR against the repo with the skeleton/base implementation of the docs here:
https://github.com/dooboolab/react-native-iap/pull/1366

**Comment by hyochan** [comment]
We now have a doc site in [react-native-iap.dooboolab.com](https://react-native-iap.dooboolab.com) :tada:
